spinta/commands/write.py

```diff
@@ -550,11 +550,16 @@ async def read_existing_data(
         # When updating by id only, there must be exactly one existing record.
         if data.action == Action.UPSERT or _has_id_in_where(data.given):
             rows = list(itertools.islice(rows, 2))
-            if len(rows) == 1:
+            number_of_rows = len(rows)
+            if number_of_rows == 1:
                 data.saved = rows[0]
                 data.saved['_type'] = data.model.model_type()
-            elif len(rows) > 1:
-                data.error = exceptions.MultipleRowsFound(data.model, _id=rows[0]['_id'])
+            elif number_of_rows > 1:
+                data.error = exceptions.MultipleRowsFound(
+                    data.model,
+                    _id=rows[0]['_id'],
```

Comment:
Only taking the `_id` from the first row?

Reply:
yes, because if multiple rows found under same `_id` - all of them will have same `_id`